router/src/server.rs

```diff
@@ -397,6 +404,15 @@ async fn generate(
         time_per_token.as_millis().to_string().parse().unwrap(),
     );
 
+    headers.insert(
+        "x-predibase-model-id",
```

Comment:
Nit: Might be a bit strange to hard-code predibase-specific things into an open-source library.

Reply:
Agree. Can we just return the base model name / size?

Reply:
sure can just make it model-id and adapter-id

Reply:
Convection is still to use X for custom headers so would something like `x-lorax-model=pb://model-name` make sense and cater for HF models too?